guides/common/modules/proc_adding-custom-rpm-repositories.adoc

```diff
@@ -1,57 +1,66 @@
-[[Adding_custom_RPM_Repositories]]
+[id="Adding_Custom_RPM_Repositories_{context}"]
 = Adding {customrpmtitle} Repositories
 
 Use this procedure to add {customrpm} repositories in {Project}.
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-rpm-repositories[].
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-rpm-repositories[].
 
 The Products window in the {ProjectWebUI} also provides a *Repo Discovery* function that finds all repositories from a URL and you can select which ones to add to your {customproduct}.
 For example, you can use the *Repo Discovery* to search `http://yum.postgresql.org/9.5/redhat/` and list all repositories for different {RHEL} versions and architectures.
-This helps users save time importing multiple repositories from a single source.
+This helps users save time importing multiple YUM repositories from a single source.
```

Comment:
Should it be Yum? I am pretty sure that is the standard?

Reply:
Should it be `RPM repositories` or `repositories containing RPM packages`?

I've made the change to move away from the default "repository" = "repositories for EL containing rpm packages".

Reply:
Can we not change it for now and we can go and answer your questions at a later stage. I don't want to block this PR but I don't know the answers? If we don't change, and look through this later?

Reply:
Reverted for now.